ultravox/training/train.py

```diff
@@ -48,16 +48,20 @@ def prepare_dataset(
     stop_strategy: datasets.StopStrategy,
     num_samples: Optional[int] = None,
     include_alt_fields: bool = False,  # whether to generate tensors for text-only input (e.g., used for KD training)
+    is_val_set: bool = False,
```

Comment:
The name and the function of this variable don't quite match and are confusing to me.

Just to tell you what I mean, I think a name like `enforce_ds_len_in_epoch_mode` would better describe what this variable wants to achieve (though it's too long). Also, the default of False would work better in that case since you only need to enable it when it's needed instead of opting out.